tests/server/metrics-errors.js

```diff
@@ -36,18 +36,29 @@ define([
     }, dfd.reject.bind(dfd)));
   };
 
-  suite['#get /metrics-errors - returns 200 without query'] = function () {
```

Comment:
I was thinking for the people that load fxa on the last train that uses the GET endpoint and then keep the window open, which seems to happen regularly.